<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameResultDialog</class>
 <widget class="QDialog" name="GameResultDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Результат игры</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="icon_label">
     <property name="styleSheet">
      <string notr="true">font-size: 60px; margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="styleSheet">
      <string notr="true">font-size: 26px;
            font-weight: bold;
            color: {title_color};
            padding: 5px;
            margin-bottom: 5px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="message_label">
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
color: #666;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_play">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
        background-color: #FFFFFF;
        border: 3px solid #7E57C2;
        border-radius: 36px;
        font-size: 34px;
        color: #7E57C2;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #EDE7F6;
        border: 3px solid #5E35B1;
        color: #5E35B1;
    }
    QPushButton:pressed {
        background-color: #D1C4E9;
        border: 3px solid #4527A0;
        color: #4527A0;
    }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
